core/src/main/scala/org/broadinstitute/dsde/rawls/genomics/GenomicsService.scala

```diff
@@ -24,7 +24,8 @@ class GenomicsService(protected val ctx: RawlsRequestContext,
 )(implicit protected val executionContext: ExecutionContext)
     extends RoleSupport
     with FutureSupport
-    with UserWiths {
+    with UserWiths
+    with GenomicsServiceRequest{
```

Comment:
supernit: missing space before the curlies -- another thing scalafmt might take care of for you?